src/Time/Actions/UpdateU.hpp

```diff
@@ -26,6 +26,28 @@ class GlobalCache;
 // IWYU pragma: no_forward_declare db::DataBox
 /// \endcond
 
+/// Perform variable updates for one substep
```

Comment:
should this be `(sub)step`? Since I guess for LMM steppers it updates for one step?